controllers/splits.ts

```diff
@@ -97,16 +204,19 @@ const update_split = asyncHandler(async (req, res, next) => {
   });
 
   if (!splitId) {
+    console.log("split not found");
```

Comment:
If we need to keep this, let's switch to logger.

```suggestion
    log.debug("split not found");
```